components/apiKey/APIKeys.tsx

```diff
@@ -38,9 +38,10 @@ const APIKeys = ({ team }: APIKeysProps) => {
     setSelectedApiKey(null);
     setConfirmationDialogVisible(false);
 
+    const { error } = (await response.json()) as ApiResponse;
```

Comment:
This should be inside the if block below.

Reply:
added in the "refactor: add error destructuring into the if statement" commit.